commands.js

```diff
@@ -1187,7 +1187,7 @@ var commands = exports.commands = {
 			target = target.substr(0, commaIndex);
 		}
 		targetUser = Users.get(target);
-		if (!targetUser) return this.sendReply("User '" + this.targetUsername + "' not found.");
+		if (!targetUser) return this.sendReply("User '" + (targetUser ? targetUser.name : target) + "' not found.");
```

Comment:
Um, this is in a `if (!targetUser)` block, so `targetUser` is guaranteed to be falsy. This is equivalent to:

    if (!targetUser) return this.sendReply("User '" + target + "' not found.");